### PR TITLE
Social shares on Buyer's Guide product pages

### PR DESCRIPTION
**Pull request by mmmavis**
Related to #1899 

Covers
- social shares on Buyer's Guide product pages

Based on specs in https://github.com/mozilla/foundation.mozilla.org/issues/1899#issuecomment-431146630 but we might want to update the pre-populated text a bit though... /cc @kristinashu

### CODE REVIEW

**Comment by kristinashu** [comment]
Amazing! You got it to pull the product name and it's creepiness!

File: source/js/buyers-guide/components/creep-vote/creep-vote.jsx [hunk 1]
```diff
@@ -159,6 +160,18 @@ export default class CreepVote extends React.Component {
    * @returns {jsx} What users see when they have voted on this product.
    */
   renderDidVote(){
+    let numGroups = 5;
```
Comment (alanmoo) [inline source/js/buyers-guide/components/creep-vote/creep-vote.jsx]: Pomax landed a PR that has the creepiness groups as a constant- can we leverage that so we're not replicating it here?

Comment (mmmavis) [reply source/js/buyers-guide/components/creep-vote/creep-vote.jsx]: @alanmoo can you point me where that variable is? I can't find it in `creep-vote.jsx`

Comment (alanmoo) [reply source/js/buyers-guide/components/creep-vote/creep-vote.jsx]: https://github.com/mozilla/foundation.mozilla.org/pull/1967/files (creepiness-labels.js)

File: source/js/buyers-guide/components/creep-vote/creep-vote.jsx [hunk 2]
```diff
@@ -159,6 +160,18 @@ export default class CreepVote extends React.Component {
    * @returns {jsx} What users see when they have voted on this product.
    */
   renderDidVote(){
+    const NUM_GROUPS = CREEPINESS_LABELS.length;
+    let userVoteGroup = Math.floor(this.state.creepiness/(100/NUM_GROUPS));
+    let creepType;
+
+    if (userVoteGroup < Math.floor(NUM_GROUPS/2)) { // lower half groups
```
Comment (alanmoo) [inline source/js/buyers-guide/components/creep-vote/creep-vote.jsx]: Can't this be something like

`creepType = CREEPINESS_LABELS[userVoteGroup]`?

Comment (mmmavis) [reply source/js/buyers-guide/components/creep-vote/creep-vote.jsx]: Nah, it's not 1-1. We only have 3 versions of keyword we wanna share in social share.

Comment (mmmavis) [reply source/js/buyers-guide/components/creep-vote/creep-vote.jsx]: > version depending on a users vote: "CREEPY" (first two bars), "NOT REALLY CREEPY" (middle), "NOT CREEPY" (last two bars)]

Comment (kristinashu) [reply source/js/buyers-guide/components/creep-vote/creep-vote.jsx]: We can do what Pomax did to make things consistent.

Comment (mmmavis) [reply source/js/buyers-guide/components/creep-vote/creep-vote.jsx]: k I'll update - @kristinashu should I make the label all caps in social share? e.g., SOMEWHAT CREEPY, A LITTLE CREEPY etc

Comment (kristinashu) [reply source/js/buyers-guide/components/creep-vote/creep-vote.jsx]: lol yeah uppercase is great, thank you!

Comment (kristinashu) [reply source/js/buyers-guide/components/creep-vote/creep-vote.jsx]: Unless that's more complicated, then lowercase is cool too!

**Comment by mmmavis** [comment]
Done! https://github.com/mozilla/foundation.mozilla.org/pull/1975 @alanmoo

**Pull request merged**